examples/megacmd/megacmdexecuter.cpp

```diff
@@ -3013,9 +3047,17 @@ void MegaCmdExecuter::executecommand(vector<string> words, map<string, int> *clf
     }
     else if (words[0] == "lpwd")
     {
-        string cCurrentPath = fsAccessCMD->getCurrentLocalPath();
+        string relativePath = ".";
+        string absolutePath = "Unknown";
+        string localRelativePath;
+        fsAccessCMD->path2local(&relativePath, &localRelativePath);
+        string localAbsolutePath;
+        if (fsAccessCMD->expanselocalpath(&localRelativePath, &localAbsolutePath))
+        {
+            fsAccessCMD->local2path(&localAbsolutePath, &absolutePath);
+        }
```

Comment:
Wouldn't this be something we could encapsulate in file access. Leaving sth like:
string localCurrentPath;
string currentPath;
fsAccessCMD->getCurrentPath(&localCurrentPath);
fsAccessCMD->local2path(&localCurrentPath, &currentPath);
if (!currentPath.size())
{
currentPath = "Unknown;
}